src/pip/_internal/commands/debug.py

```diff
@@ -197,6 +193,7 @@ class DebugCommand(Command):
     ignore_require_venv = True
 
     def __init__(self, *args, **kw):
+        # type: (*Any, **Any) -> None
```

Comment:
Do we ever initialize commands with anything other than `WhateverCommand()`? If so, I think we'd want to make a blanket PR that removes these unnecessary `(*args, **kwargs)`.

Reply:
> Do we ever initialize commands with anything other than WhateverCommand()? If so, I think we'd want to make a blanket PR that removes these unnecessary (*args, **kwargs).

I might have to look more but I think we only use a limited set of function parameters. Explicity using them instead of blanket `(*args, **kwargs).` is a good idea. I will make an issue for this to track the subsequent PRs

Reply:
https://github.com/pypa/pip/issues/8244 created for this change.